packages/combo-box/test/scrolling.test.js

```diff
@@ -74,18 +73,6 @@ describe('scrolling', () => {
       expect(scroller.scrollTop).to.equal(0);
     });
 
-    function expectSelectedItemPositionToBeVisible() {
```

Comment:
Usage of this function has been removed in #6055 where we removed corresponding logic.